packages/cache-manager/index.ts

```diff
@@ -5,16 +5,16 @@ export type IDataProviderCallback = () => string | Promise<string>;
 
 export interface CacheManager {
     getItem: (key: string) => Promise<string>;
-    setItem: (key: string, value: string) => Promise<void>;
+    setItem: (key: string, value: string, ttl:number) => Promise<void>;
```

Comment:
you should use the third argument as the options object, and not the TTL directly.